lib/compute/network.js

```diff
@@ -215,6 +221,137 @@ Network.prototype.createFirewall = function(name, config, callback) {
+ *    subnetwork. For example, 10.0.0.0/8 or 192.168.0.0/16. Ranges must be
+ *    unique and non-overlapping within a network.
+ * @param {string} config.region - URL of the region where the Subnetwork
+ *    resides.
+ * @param {function} callback - The callback function.
+ * @param {?error} callback.err - An error returned while making this request.
+ * @param {module:compute/rule} callback.rule - The created Rule object.
+ * @param {module:compute/operation} callback.operation - An operation object
+ *     that can be used to check the status of the request.
+ * @param {object} callback.apiResponse - The full API response.
+ *
+ * @example
+ * var name = 'new-subnetwork-name';
+ *
+ * var config = {
+ *   region : 'regions/us-east1',
+ *   ipCidrRange: '10.0.1.0/24'
+ * };
+ *
+ * network.createSubnetwork(name, config,
+ *  function (err, subnetwork, operation, apiResponse) {
+ *   // `subnetwork` is a Subnetwork object.
+ *
+ *   // `operation` is an Operation object that can be used to check the status
+ *   // of the request.
+ * });
+ */
+Network.prototype.createSubnetwork = function(name, config, callback) {
+  config = extend({}, config, {
+    network: this.formattedName
+  });
+
+  var region = new Region(this.compute, config.region);
+  delete config.region;
+
+  return region.createSubnetwork(name, config, callback);
+};
+
+/**
+ * Get a list of subnetworks in this network.
+ *   * @resource [Subnetworks Overview]{@link https://cloud.google.com/compute/docs/subnetworks}
+ * @resource [Subnetworks: list API Documentation]{@link https://cloud.google.com/compute/docs/reference/latest/subnetworks}
+ *
+ * @param {object=} options - Subnetwork search options.
+ * @param {boolean} options.autoPaginate - Have pagination handled
+ *     automatically. Default: true.
+ * @param {string} options.filter - Search filter in the format of
+ *     `{name} {comparison} {filterString}`.
+ *     - **`name`**: the name of the field to compare
+ *     - **`comparison`**: the comparison operator, `eq` (equal) or `ne`
+ *       (not equal)
+ *     - **`filterString`**: the string to filter to. For string fields, this
+ *       can be a regular expression.
+ * @param {number} options.maxApiCalls - Maximum number of API calls to make.
+ * @param {number} options.maxResults - Maximum number of firewalls to return.
+ * @param {string} options.pageToken - A previously-returned page token
+ *     representing part of the larger set of results to view.
+ * @param {function} callback - The callback function.
+ * @param {?error} callback.err - An error returned while making this request.
+ * @param {module:compute/firewall} callback.subnetworks - Subnetwork objects
+ *     from your project.
+ * @param {?object} callback.nextQuery - If present, query with this object to
+ *     check for more results.
+ * @param {object} callback.apiResponse - The full API response.
+ *
+ * @example
+ * gce.getSubnetworks(function(err, subnetworks) {
```

Comment:
These examples should show `network.getSubnetworks` instead of `gce.getSubnetworks`